integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -0,0 +1,427 @@
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db import models
+from django.db.models import Q
+from django.utils.translation import ugettext_lazy as _
+
+from linkcheck.models import Link
+
+from .languages.language import Language
+from ..constants import status
+from ..utils.translation_utils import ugettext_many_lazy as __
+
+
+class AbstractContentTranslation(models.Model):
+    """
+    Data model representing a translation of some kind of content (e.g. pages or events)
+    """
+
+    title = models.CharField(max_length=1024, verbose_name=_("title"))
+    slug = models.SlugField(
+        max_length=1024,
+        allow_unicode=True,
+        verbose_name=_("URL parameter"),
+        help_text=__(
+            _("String identifier without spaces and special characters."),
+            _("Unique per region and language."),
+            _("Leave blank to generate unique parameter from title."),
+        ),
+    )
+    #: Manage choices in :mod:`~integreat_cms.cms.constants.status`
+    status = models.CharField(
+        max_length=9,
+        choices=status.CHOICES,
+        default=status.DRAFT,
+        verbose_name=_("status"),
+    )
+    content = models.TextField(blank=True, verbose_name=_("content"))
+    language = models.ForeignKey(
+        Language,
+        on_delete=models.CASCADE,
+        verbose_name=_("language"),
+    )
+    currently_in_translation = models.BooleanField(
+        default=False,
+        verbose_name=_("currently in translation"),
+        help_text=_(
+            "Flag to indicate a translation is being updated by an external translator"
+        ),
+    )
+    version = models.PositiveIntegerField(default=0, verbose_name=_("revision"))
+    minor_edit = models.BooleanField(
+        default=False,
+        verbose_name=_("minor edit"),
+        help_text=_(
+            "Tick if this change does not require an update of translations in other languages."
+        ),
+    )
+    last_updated = models.DateTimeField(
+        auto_now=True,
+        verbose_name=_("modification date"),
+    )
+    creator = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        verbose_name=_("creator"),
+    )
+    links = GenericRelation(Link, related_query_name="poi_translations")
```

Comment:
```suggestion
    links = GenericRelation(Link)
```
This should probably be removed, right?